kolibri/content/serializers.py

```diff
@@ -329,16 +331,21 @@ def to_representation(self, instance, progress_fraction=None, annotate_progress_
         return value
 
     def get_num_coach_contents(self, instance):
-        # TODO return 0 by default if user is logged in as Learner
-        if instance.kind == content_kinds.TOPIC:
-            # get_descendants is weird for channels
-            return instance.get_descendants() \
-                .filter(coach_content=True, available=True) \
-                .exclude(kind=content_kinds.TOPIC) \
-                .distinct() \
-                .count()
-        else:
-            return 1 if instance.coach_content else 0
+        user = self.context["request"].user
+        if user.is_facility_user:  # must be facility user
```

Comment:
this check is to exclude `AnonymousUser`?

Reply:
yes